siteimprove/admin/js/siteimprove.js

```diff
@@ -105,7 +105,7 @@
           
           //Scroll to the target element
           $([document.documentElement, document.body]).animate({
-            scrollTop: $(".si-highlight").offset().top
+            scrollTop: $(".si-highlight").offset().top - $("#wpadminbar").height()
```

Comment:
Couldn't you use the `scrollIntoView` function - like they suggest in this article: https://stackoverflow.com/questions/23062146/javascript-scroll-and-focus. This seems a little "manual" in terms of calculating screen offsets etc...